src/frontends/tensorflow_lite/src/graph_iterator_flatbuffer.hpp

```diff
@@ -58,6 +61,14 @@ class GraphIteratorFlatBuffer {
 
     /// Return Decoder for the current node that iterator points to
     std::shared_ptr<ov::frontend::tensorflow_lite::DecoderFlatBuffer> get_decoder() const;
+
+    /// \brief Returns the number of sub-graphs that can be enumerated with get_subgraph
+    size_t get_subgraph_size() const;
+
+    /// \brief Returns iterator for a subgraph created on demand
+    /// If there is no query for specific sub-graph iterator shouldn't be created
+    /// idx should be in range 0..get_subgraph_size()-1
+    std::shared_ptr<GraphIteratorFlatBuffer> get_subgraph(const size_t& idx) const;
```

Comment:
do you have the common cache of already converted sub-graph accessible simultaneously to all sub-graphs?

Reply:
no I don't. there is no such use case in known TFL models